sig/lrama/grammar/rule_builder.rbs

```diff
@@ -7,26 +7,35 @@ module Lrama
       attr_reader user_code: Lexer::Token::UserCode?
       attr_reader precedence_sym: Lexer::Token?
 
+      @rule_counter: Counter
+      @midrule_action_counter: Counter
+      @position_in_original_rule_rhs: Integer?
+      @skip_preprocess_references: bool?
```

Comment:
This is not optional, then `bool` (without `?`) is correct.

Reply:
Indeed it is. I updated this PR/